template-detail-communities.php

```diff
@@ -308,6 +308,20 @@ function ( $field ) {
 		}
 	}
 
+	/**
+	 * Contact form box
+	 * (actually it's a Gravity Forms form)
+	 * ----------------------------- */
+	$metabox_contactform_id = get_field( 'metabox_contactform_id' );
+	if ( $metabox_contactform_id ) {
```

Comment:
Helemaal superduper. 1 klein aanvulling: willen we checken of Gravityforms (en dus het block daarvoor) beschikbaar is?